ideascube/tests/test_views.py

```diff
@@ -511,3 +517,55 @@ def test_by_tag_page_is_paginated(app, monkeypatch):
     assert not response.pyquery.find('.next')
     assert response.pyquery.find('.previous')
     response = app.get(url + '&page=3', status=404)
+
+
+def test_build_package_card_info_must_no_fail_for_no_package(app, systemuser):
+    from ideascube.configuration import set_config
+    from ideascube.views import build_package_card_info
+    set_config('home-page', 'displayed-package-ids', ['test.package1'], systemuser)
+    assert build_package_card_info() == []
+
+
+def test_build_package_card_info(app, systemuser, catalog):
```

Comment:
`build_package_card_info` is a internal function only used by the views.

I don't know where to put the function elsewhere to views.py and so, I test it in test_views.py.
If you have an idea, I take it :smile: